inspire/base/templates/header.html

```diff
@@ -31,11 +28,12 @@
 ## structure: (endpoint, values, blueprint, caption)
 #}
 {% set navigation_bar = [
-  ('search.index', {}, 'search', 'Home'),
-  ('webdeposit.create', {"deposition_type": "literature"}, 'webdeposit', 'Submit'),
-  ('/inspire/help', {}, '', 'Help'),
+    ('search.index', {}, 'search', 'Home'),
```

Comment:
can you leave the indentation as it was so that git does not detect it as a change?